configuration/driver/icedrv_forcing.F90

```diff
@@ -20,62 +21,64 @@ module icedrv_forcing
          fsnow, sst, sss, uocn, vocn, qdp, hmix, Tf, opening, closing, sstdat
+#ifdef USE_NETCDF
+      use netcdf
+#endif
 
       implicit none
       private
+
       public :: init_forcing, get_forcing, interp_coeff, &
                 interp_coeff_monthly, get_wave_spec
 
-      integer (kind=int_kind), parameter :: &
-         ntime = 8760        ! number of data points in time
+      integer (kind=int_kind) :: &
+         ntime               ! number of data points in time
 
       integer (kind=int_kind), public :: &
          ycycle          , & ! number of years in forcing cycle
          fyear_init      , & ! first year of data in forcing cycle
          fyear           , & ! current year in forcing cycle
          fyear_final         ! last year in cycle
 
-      real (kind=dbl_kind), dimension(ntime) :: &
-            fsw_data, & ! field values at temporal data points
-           cldf_data, &
-          fsnow_data, &
-           Tair_data, &
-           uatm_data, &
-           vatm_data, &
-           wind_data, &
-          strax_data, &
-          stray_data, &
-           rhum_data, &
-             Qa_data, &
-           rhoa_data, &
-           potT_data, &
-            flw_data, &
-            qdp_data, &
-            sst_data, &
-            sss_data, &
-           uocn_data, &
-           vocn_data, &
-          frain_data, &
-          swvdr_data, &
-          swvdf_data, &
-          swidr_data, &
-          swidf_data, &
-           zlvl_data, &
-           hmix_data
+      real (kind=dbl_kind), allocatable :: &
+            fsw_data(:), & ! field values at temporal data points
+           cldf_data(:), &
+          fsnow_data(:), &
+           Tair_data(:), &
+           uatm_data(:), &
+           vatm_data(:), &
+           wind_data(:), &
+          strax_data(:), &
+          stray_data(:), &
+           rhum_data(:), &
+             Qa_data(:), &
+           rhoa_data(:), &
+           potT_data(:), &
+            flw_data(:), &
+            qdp_data(:), &
+            sst_data(:), &
+            sss_data(:), &
+           uocn_data(:), &
+           vocn_data(:), &
+          frain_data(:), &
+          swvdr_data(:), &
+          swvdf_data(:), &
+          swidr_data(:), &
+          swidf_data(:), &
+           zlvl_data(:), &
+           hmix_data(:), &
+           open_data(:), &
+           clos_data(:)
 
       real (kind=dbl_kind), dimension(nx) :: &
           sst_temp
 
-      real (kind=dbl_kind), dimension(ntime) :: &
-           open_data, &
-           clos_data
-
       character(char_len), public :: &
          atm_data_format, & ! 'bin'=binary or 'nc'=netcdf
          ocn_data_format, & ! 'bin'=binary or 'nc'=netcdf
          bgc_data_format, & ! 'bin'=binary or 'nc'=netcdf
-         atm_data_type,   & ! 'default', 'clim', 'CFS'
-         ocn_data_type,   & ! 'default', 'SHEBA'
+         atm_data_type,   & ! 'default', 'clim', 'CFS', 'MOSAiC'
```

Comment:
Thanks for the suggestions Anton! @eclare108213 and @apcraig, what would you think about making the ```atm_data_type``` and ```ocn_data_type``` for this 'MDF' instead of 'MOSAiC'? Then I would also propose that we make the default path for these forcing data: ```/icepack-dirs/input/Icepack_data/forcing/MDF```.

Reply:
MDF works for me.  But how will you distinguish between MOSAiC MDF data and some other kind of MDF data?

Reply:
I believe the only place where it matters to the code that this is MOSAiC MDF data is when we check if the simulation time includes the MOSAiC leg 4 to leg 5 ship repositioning. For that check, we can check whether the forcing filename contains the string 'MOSAiC'.

Reply:
Gah - thats an annoying quirk. I don't have a straightforward answer to that. This sounds like a different measurement site though? So should the experiment start again at that point with different inititial conditions ?

Reply:
Should we have two different mosaic options.  MOSAIC123 and MOSAIC45 for instance or some other name.  These would be two separate forcing/ic options?

Reply:
I think that's a good idea. I have not worked much with the leg 5 (August-September near the North Pole) data but I will inquire about how we could have an appropriate initial condition.

Based on this discussion, I think it would be cleanest if the data archive contained separate files for legs 1-4 (October-July along transpolar drift) and leg 5 (August-September near the North Pole). I will inquire within the MOSAiC Model Forcing Dataset working group to see whether others agree and if we can update the data archive.